EXILED/Exiled.API/Features/Roles/IVoiceRole.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="IVoiceRole.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features.Roles
+{
+    using PlayerRoles.Voice;
+
+    /// <summary>
+    /// Interface for all roles with <see cref="VoiceModuleBase"/>.
+    /// </summary>
+    public interface IVoiceRole
+    {
+        /// <summary>
+        /// Gets the <see cref="VoiceModuleBase"/> of the role.
+        /// </summary>
+        public VoiceModuleBase VoiceModule { get; }
+    }
+}
```

Comment:
i don't thinks it's needed to create the same interface than NW PlayerRoles.Voice.IVoiceRole

Reply:
It seems to me that if weve already created a lot of our own entities, then the interfaces should be ours as well

Reply:
I agree with Irackle. We already have 2 interfaces when NW has absolutly same

Reply:
Irackle😭